test/data/issue_1959_poc.xmp

```diff
@@ -33,7 +33,6 @@
    exif:ExifVersion="0232"
    exif:FlashpixVersion="0100"
    exif:ColorSpace="65535"
-   photoshop:DateCreated="2022-01-04T09:41:01+00:00"
```

Comment:
Interesting, you took the same approach I wanted to took here: #2079.

For me it is fine to recover the test case like this.

Reply:
Great minds think alike!

it's not really a fix.  It's evasion.  It's the metadata convertors.  I don't know who thought of adding convertors to the code base.  If those convertors are not defined in a standard, they should be implemented in another different project which might use libexiv2.  However, I don't know why this is in the metadata engine.